src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -12,170 +12,113 @@ parameters:
 
   # The following parameters aren't expected to be passed in rather they are used for encapsulation
   # -----------------------------------------------------------------------------------------------
-  buildJobTimeout: 300
   centOS7Container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-source-build-20210714125450-5d87b80
   centOS8Container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-source-build-20211118190102-9355e7b
   centOSStream9Container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9-20220107135047-4cd394c
   debian9Container: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-stretch-20211001171226-047508b
+  debian9Arm64Container: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-arm64v8-20220405201138-a251961
   fedora33Container: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33-20210222183538-031e7d2
   ubuntu1804Container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-20210924170306-047508b
-  poolInternal:
+  poolInternalAmd64:
     name: NetCore1ESPool-Svc-Internal
     demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
-  poolPublic:
+  poolInternalArm64:
+    name: Docker-Linux-Arm-Internal
```

Comment:
@crummel - do we have any arm64 RHEL agents w/Docker installed that can be used here?  I like the Docker pool because machine management feels more manageable.

Reply:
There are two of them.  I think they are probably less reliable than the Docker pool and support using the Docker pool as well since there's no RHEL requirement here.